wesnoth-nr.po

```diff
@@ -3249,41 +3132,39 @@ msgstr "Proč sem přicházíš?"
 
 #. [message]: speaker=Tallin
 #: data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg:2089
-#, fuzzy
-#| msgid "We seek the lich Malifor."
 msgid "We seek the destruction of the lich Malifor."
-msgstr "Hledáme liche jménem Malifor."
+msgstr "Hledáme to něco, co zničí liche jménem Malifor."
 
 #. [message]: speaker=Abhai
 #: data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg:2094
 msgid ""
 "(<i>pales if possible</i>) Malifor...! Finally someone here in force to deal "
 "with that menace!"
 msgstr ""
+"(<i>zbledne, pokud se u ducha dá poznat</i>) Malifora...! Konečně někdo "
```

Comment:
mě se líbí víc
```suggestion
"(<i>zbledne, pokud je to u ducha možné</i>) Malifora...! Konečně někdo "
```

Reply:
Zapomněl jsem na slovo TO. Ale klidně i takto.